packages/fullscreen/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "main": "lib/index.js",
```

Comment:
Some of the fields in the package.json are missing. Can you look at the other hooks as example?